README.md

```diff
@@ -1,148 +1,188 @@
+the volume, complexity, or type of data. It ingests millions of records per
+second for time series setups and delivers analytics results in sub-second real
+time.
 
-We solve your data scaling problems and make administration a breeze.
+Crate comes with a distributed sort and aggregation engine, fast multi index
+queries, native full-text search and super simple scalability with sharding and
+partitioning builtin. Preconfigured replication takes care of data resiliency.
+The cluster management can be supervised with a built-in admin UI. Crate's
+masterless architecture and simplicity make the data part of Docker environments
+easy and elegant.
 
-**Crate.IO - Put your data to work. Simply.**
+Crate provides several installation packages, including a supported Docker
+image. It fits perfectly into an orchestrated microservices environment. It acts
+like an ephemeral, omnipresent, persistent layer for data. Application
+containers access their data regardless of which host the data nodes run.
 
-[http://crate.io][3]
+[Crate](https://crate.io/)
 
 
-# Crate Dockerfile
+![logo](https://raw.githubusercontent.com/docker-library/docs/2517900006ae5f4c03c1d43235930c59f4614394/crate/logo.png)
 
-This repository contains **Dockerfile** of [Crate][3] for [Docker][1]'s [automated build][2]
-published to the public [Docker Hub Registry][4].
+## How To Use This Image
 
+To form a cluster, start a few instances of the Crate container as a background
+daemon:
 
-## Base Docker Image
+```console
+# docker run -d crate/crate
+```
 
-- [alpine:latest][5]
+To access the admin UI, map port 4200 and point your browser to port tcp/4200 of
+a node of your choice while you start it or look up its IP later:
 
-## Installation
+```console
+# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate/crate)):4200/admin"
+```
 
-1. Install [Docker][1]
+For production use it's strongly recommended to use only one container per
+physical machine. So mapping Crate's (default) ports 4200 (HTTP) and 4300
+(Transport protocol) is a good way to provide easy access.
 
-2. Download latest automated build from public [Docker Hub Registry][2]
+```console
+# docker run -d -p 4200:4200 -p 4300:4300 crate/crate
+```
 
-    docker pull crate/crate:latest
+## Attach Persistent Data Directory
 
-Alternatively you can build an image from `Dockerfile`:
+Crate stores all important data in _/data_. To attach it to a backup service,
+map the directory to the host:
 
-    docker build -t="crate/crate" github.com/crate/docker-crate
+```console
+# docker run -d -v <data-dir>:/data crate/crate
+```
 
-## How to use this image
+**Note**: that there are way more sophisticated ways to [backup data with
+builtin commands like `CREATE
+SNAPSHOT`](https://crate.io/a/backing-up-and-restoring-crate/).
 
-    docker run -d -p 4200:4200 -p 4300:4300 crate/crate:latest
+## Use Custom Crate Configuration
```

Comment:
that makes sense - but then you go into deriving from the crate container? i think it's about passing a config file.